plugins/netmesh/plugin_impl_netmesh.go

```diff
@@ -81,17 +78,13 @@ func (plugin *Plugin) Init() error {
 // the kvdbsync is fully initialized and ready for publishing when a k8s
 // notification comes.
 func (plugin *Plugin) AfterInit() error {
-	netmeshdp = nsmdp.NewNSMDevicePlugin()
-	netmeshdp.Serve()
+	err := nsmdp.NewNSMDevicePlugin()
```

Comment:
What if we passed in the Plugin() here so we could make use of the logger from the Plugin() in the nsmdp code? That would give us consistency around logging. Thoughts?

Reply:
I thought about it once I added refactored code, I will look into it, thanks for suggestions.

Reply:
Great, thanks! We may need to come up with a broader logging strategy, so if it becomes too challenging, feel free to open an issue and we'll tackle it there.